### PR TITLE
Added Build Information to README.md

### PR DESCRIPTION
**Pull request by kathweaver**
These are frequently asked questions to people who are new to the
project and want to contribute.

### CODE REVIEW

**Comment by kathweaver** [comment]
Learned something not to do :-)

File: README.md [hunk 1]
```diff
@@ -55,6 +55,13 @@ this open source fork of Live Writer formed and ready to ship. In December 2015
 to the .NET Foundation and this passionate group of volunteer engineers rapidly assembled the first open source
 version.
 
+### Building
+OpenLiveWriter can be built by running build.cmd found in this directory
+It can be opened in Visual Studio.  The solution is in src/managed/writer.sln -- if you see errors in Visual Studio
+run build.cmd from the command prompt and it should be resolved.
```
Comment (ScottIsAFool) [inline README.md]: New lines here when there shouldn't be.

**Comment by kathweaver** [comment]
Fixed.

File: README.md [hunk 2]
```diff
@@ -55,6 +55,12 @@ this open source fork of Live Writer formed and ready to ship. In December 2015
 to the .NET Foundation and this passionate group of volunteer engineers rapidly assembled the first open source
 version.
 
+### Building
+OpenLiveWriter can be built by running build.cmd found in this directory.   
+It can be opened in Visual Studio.  The solution is in src/managed/writer.sln -- if you see errors in Visual Studio run build.cmd from the command prompt and it should be resolved.
+The main program is src/managed/OpenLiveWriter/ApplicationMain.cs .
+To run from Visual Studio, set the startup property to OpenLiveWriter.
```
Comment (ScottIsAFool) [inline README.md]: "startup project" not property

**Comment by kathweaver** [comment]
Still having git issues, closing and started this again.

**Pull request closed without merging**